dojo/finding/views.py

```diff
@@ -804,6 +804,10 @@ def edit_finding(request, fid):
             # any existing finding should be updated
             push_to_jira = push_to_jira and not push_group_to_jira and not new_finding.has_jira_issue
 
+            if old_finding.duplicate and not new_finding.duplicate:
+                reset_finding_duplicate_status_internal(request.user, new_finding.id)
+                new_finding.refresh_from_db()
+
             new_finding.save(push_to_jira=push_to_jira)
```

Comment:
The behaviour I think is caused by the dedupe running on save. So if you do NOT want dedupe to run on save, the only change needed should be to supply `dedupe_option=False` to save() here. But maybe that's not what everyone wants, so  there could be a "Run deduplication" checkbox on screen, similar to what we have for "push to JIRA".

If we don't want that, and see this "mark no longer as duplicate" as a special case we should handle, I think we could suffice with adding `dedupe_option=False` for that scenario only?

Reply:
Yes, specifying the `dedupe_option=False` makes it work as expected.

For these manual overrides, we should have a way to lock them, so that future occurrences of deduplication (e.g. running `manage.py dedupe`) do not go in a revert the state to whatever it was.